projects/packages/videopress/src/class-initializer.php

```diff
@@ -228,6 +228,14 @@ public static function register_videopress_video_block() {
 			)
 		);
 
+		wp_enqueue_script(
+			self::JETPACK_VIDEOPRESS_VIDEO_VIEW_HANDLER . '-iframe-api',
+			'https://wordpress.com/wp-content/plugins/video/assets/js/videojs/videopress-iframe-api.js',
```

Comment:
I believe this should use one of our static subdomains:

```suggestion
			'https://s0.wp.com/wp-content/plugins/video/assets/js/videojs/videopress-iframe-api.js',
```